src/OpenTelemetry/ExporterOptions/IHttpClientFactoryExporterOptions.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Net.Http;
+
+namespace OpenTelemetry.Exporter
+{
+    public interface IHttpClientFactoryExporterOptions
```

Comment:
Does it make sense to just fold these options back into the exporter options? My thinking is that this option is unlike the processor/metric reader options in that it actually is about the exporter itself.